test/unit/package_test.rb

```diff
@@ -1,68 +1,403 @@
 require 'test_helper'
 
 class PackageTest < Minitest::Test
-  include ActiveShipping::Test::Fixtures
-
-  GRAMS_IN_AN_OUNCE = 28.349523125
-  OUNCES_IN_A_GRAM = 0.0352739619495804
-  INCHES_IN_A_CM = 0.393700787401575
-  CM_IN_AN_INCH = 2.54
-
   def setup
-    @imperial_package = Package.new(65, [3, 6, 8.5],
-                                    :units => :imperial,
-                                    :value => 10.65,
-                                    :currency => 'USD'
-                        )
-
-    @metric_package = Package.new(100, [5, 18.5, 40],
-                                  :value => 860,
-                                  :currency => 'CAD'
-                      )
-  end
-
-  def test_accessors
```

Comment:
This test was incomplete and wasn't really testing everything clearly. Replaced it with many other tests testing each feature explicitly.